stam.py

```diff
@@ -1,17 +1,20 @@
 __author__ = 'Adi'
 
+FIRST_NAME = 'Adi'
+LAST_NAME = 'Kadar Levi'
 
-def main():
-    input_file = open(r'C:\..., text.txt', 'r')
-    new_file = open('new_text.txt', 'w+')
-    # line = None
-    for line in input_file:
-        new_line = 'Adi ' + line[:-1] + ' Kadar Levi'
-        print(new_line)
-        new_file.write(new_line + '\n')
 
-    input_file.close()
-    new_file.close()
+def main():
+    try:
+        input_file_path = input('please enter text file path\n')
+        with open(input_file_path, 'r') as input_file:
+            with open('new_text.txt', 'w+') as new_file:
+                for line in input_file:
+                    new_line = FIRST_NAME + ' ' + format(line[:-1]) + ' ' + LAST_NAME
```

Comment:
you should use the `format `function for the string creation instead of using `+`.
also, use `strip` for the `\n` removal

Reply:
done